### DISCUSSION TITLE
frontend reset password "The passwords do not match" error

### DISCUSSION THREAD

**Discussion by adnankaya** [fastapi/full-stack-fastapi-template#1170] [Questions] [answered]
Hi @tiangolo @alejsdev , there is an error on frontend I pointed in the following code. As a backend developer I only could find the problem and show it. Hopefully it helps frontend devs to fix it.

### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
export const confirmPasswordRules = (
  getValues: () => any,
  isRequired = true,
) => {
  const rules: any = {
    validate: (value: string) => { 
      // qwert123
      console.log(value);
      /*
      {
        "new_password": "qwert123",
        "confirm_password": "qwert123"
      }
      */ 
      console.log(getValues());
      // getValues().password is None because as we can see above log message shows only object has new_password and confirm_password keys
      return value === getValues().password || "The passwords do not match"},
  }

  if (isRequired) {
    rules.required = "Password confirmation is required"
  }

  return rules
}
```

### Description

Open browser -> Forgot Password -> Enter Email -> Continue -> Mail (password reset link) -> http://localhost/reset-password?token=eyJhbGciOiJI... -> Set Password (Entered qwert123) / Confirm Password (Entered qwert123)

The passwords do not match error appears above the Reset Password button.

### Operating System

macOS

### Operating System Details

Mac Os M2

### Python Version

Python 3.10.14

**Comment by alejsdev** [comment, marked as answer]
Hi @adnankaya, thanks for the report! :mag: 

I already replicated the error and fixed it in https://github.com/tiangolo/full-stack-fastapi-template/pull/1171. You can try again. I'll be waiting for your reply. Thanks :nerd_face:

**Reply by adnankaya** [reply]
It works! 🎉 Thank you.
